src/etc/inc/dyndns.class

```diff
@@ -450,6 +450,8 @@
 					break;
 				case 'easydns':
 					$needsIP = TRUE;
+					if (isset($this->_dnsWildcard))
+						$this->_dnsWildcard = "ON";
```

Comment:
Like other places in the code you should check if it's != "OFF" before assume it should be "ON"